modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -53,6 +54,9 @@ def coordinates
       end
 
       def facility_coordinates
+        # doesn't appear to be happening. perhaps this is no longer relevant
+        # looks like the front end doesn't send the facility id.
+        # if that's hard-coded on the FE then maybe we don't need this code
         Rails.logger.info('CC providers call get facilities', facility_id: params[:facilityId])
```

Comment:
Category: I am here with id
Action: remove. This isn't happening anyhow, presumably because the FE isn't sending the facility id which means this code is never hit.